doc/manual/src/release-notes/release-notes.md

```diff
@@ -1 +1,7 @@
 # Nix Release Notes
+
+Nix has a release cycle of roughly 6 weeks.
+Notable changes and additions are announced in the release notes for each version.
+
+Bugfixes can be backported on request to the Nix version used in the latest NixOS release, which is announced in the [NixOS release notes](https://nixos.org/manual/nixos/stable/release-notes.html#ch-release-notes).
```

Comment:
I was a bit confused by this, so I split into two sentences.

Also I wanted to include the part about no skipped backports; users should know it is safe rely on this (and we should also make sure to not forget when backporting).

Both together:
```suggestion
Bugfixes can be backported on request to previous Nix releases.
We typically backport only as far back as the Nix version used in the latest NixOS release, which is announced in the [NixOS release notes](https://nixos.org/manual/nixos/stable/release-notes.html#ch-release-notes).

Backports never skip releases (if a feature is backported to `x.y`, it must also be available in `x.(y+1)`;
This ensures that upgrading from an older version with backports is still safe and no backported functionality will go missing.
```